### ISSUE TITLE
Stopped or crashed unexpectedly due to handling backslash "\" wrongly in filename.

### ISSUE THREAD

**Issue by bluet** [jotta/jotta-cli-issues#157]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
0.11.44593
Ubuntu 20.04 64bit

**Description of problem:**
Stopped or crashed unexpectedly due to handle backslash "\" wrongly in filename.

I have a system file which has a backslash "\" in it's filename
```
$ ls snap-cano* 
'snap-canonical\x2dlivepatch-105.mount'
```

jotta-cli stopped working while handling it.
It seems that jotta-cli replaced it with slach "/" so it couldn't find the file. And then it crashed.

**Expected:**

jotta-cli treats the filename as-is and backup.

**jotta-cli status (`jotta-cli status`):**
```

```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )**
```
pid:350 2021/09/02 20:35:28 Error checksumming files.Open(/xxxx/etc/systemd/system/snap-canonical/x2dlivepatch-105.mount | /xxxx/etc/systemd/system/snap-canonical/x2dlivepatch-105.mount) : open /xxxx/etc/systemd/system/snap-canonical/x2dlivepatch-105.mount: no such file or directory
pid:350 2021/09/02 20:35:28 Error uploading {"Name":"snap-canonical\\x2dlivepatch-105.mount","Size":321,"Modified":"2021-09-02T20:16:25.11039495+08:00","Created":"2021-09-02T20:18:06.39462594+08:00","Checksum":"","State":"Added"} => files.Open(/xxxx/etc/systemd/system/snap-canonical/x2dlivepatch-105.mount | /xxxx/etc/systemd/system/snap-canonical/x2dlivepatch-105.mount) : open /xxxx/etc/systemd/system/snap-canonical/x2dlivepatch-105.mount: no such file or directory


Exiting:
------------------------------------------------------------------------------
 Account   : bluetraveler@gmail.com on Jottacloud
 Usage     : 755.66GiB / ( Unlimited )
 Device    : SolidRock
------------------------------------------------------------------------------
 Backups   :
------------------------------------------------------------------------------
   Path      : /backup/homes
   Scanning  : /xxxxx
   Files     : 133417 (469.21GiB)
------------------------------------------------------------------------------
OK
```

**Traceback**
```

```

**Additional info:**

**Comment by bluet** [comment]
@Kimbsen Thanks for the quick reply!

I really need that get fixed soon as my system is in danger.

**Comment by coopersimon** [comment]
This issue should be fixed in the newest version of jotta-cli (0.11.47628)
